backends/tofino/bf-p4c/.gdbinit

```diff
@@ -0,0 +1,728 @@
+# Copyright 2013-2024 Intel Corporation.
+#
+# This software and the related documents are Intel copyrighted materials,
+# and your use of them is governed by the express license under which they
+# were provided to you ("License"). Unless the License provides otherwise,
+# you may not use, modify, copy, publish, distribute, disclose or transmit this
+# software or the related documents without Intel's prior written permission.
+#
+# This software and the related documents are provided as is, with no
+# express or implied warranties, other than those that are expressly stated
+# in the License.
+
+# vi: ft=python
+set print static-members off
+set print object
+set unwindonsignal on
+set unwind-on-terminating-exception on
+set python print-stack full
```

Comment:
This should probably be merged with the top-level .gdbinit file -- it shares a lot in common with that in any case